### ISSUE TITLE
prompt the users to use node.js 4.0+

### ISSUE THREAD

**Issue by dabeng** [patternfly-webcomponents/patternfly-webcomponents#7]
We need to prompt the users to use node.js 4.0+, or else they will get the following error when they run "gulp serve"
![screenshot from 2016-11-29 10-37-31](https://cloud.githubusercontent.com/assets/5689075/20694436/c84aeeb0-b5aa-11e6-9b9d-ef900f83d66d.png)

**Comment by priley86** [comment]
is there a change you'd like to contribute? I'm not sure what's necessary here - an addition in the gulpfile to warn them?

**Comment by priley86** [comment]
I agree that node4/node6 errors are not as indicative as they should be. I've just gotten used to seeing es6 things like `const` and knowing it may be a node version issue (so I'll use `nvm` to change node versions locally and test this). It would nice if this weren't a mystery.
